docs/DeveloperGuide.md

```diff
@@ -312,16 +312,79 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorTrack`, the **Actor** is the `Tutor` and the **Person** is the `Tutee`, unless specified otherwise)
+
+**Use case: View all Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+
+    Use case ends.
+
+**Extensions**
+* 2a. The list is empty.
+* *a. Connection lost.
+    * *a1. System informs the user.
+
+  Use case ends.
+
+**Use case: Add a Person**
```

Comment:
If you want, this was the one i wrote previously for add use case.

**Use case: UC01 - Add a student**

**MSS**

1.  Tutor enters student's information to be added.
2.  TutorTrack adds the student's data.
3.  TutorTrack displays success message.
    Use case ends.

**Extensions**

* 2a. Required fields are left empty.

    * 2a1. TutorTrack shows an error message.    

    Use case ends.

* 2b. The given phone number is invalid.

    * 2b1. TutorTrack shows an error message.

    Use case ends.

* 2c. The given email is invalid.

    * 2c1. TutorTrack shows an error message.

    Use case ends.

* 2d. The given grades are invalid.

    * 2d1. TutorTrack shows an error message.

    Use case ends.

* 2e. The student exists in the database already.

    * 2e1. TutorTrack shows an error message.

    Use case ends.